### ISSUE TITLE
Investigating if it is possible to fix multiple broken links on fr/consignes-auteurs

### ISSUE THREAD

**Issue by anisa-hawes** [programminghistorian/jekyll#2826]
I note that ~10 links to a sub-domain of the Canadian government's website <https://www.quebec.ca/> which are included in fr/consignes-auteurs have broken...

<http://bdl.oqlf.gouv.qc.ca> is now redirecting to <https://vitrinelinguistique.oqlf.gouv.qc.ca/>

Our FR author guidelines currently link to multiple pages across that website. I'm going to take a look now, but I may need some support from the @programminghistorian/french-team to identify which replacement links will be suitable...

**Comment by spapastamkou** [comment]
It is me who had inserted these links to have further information and examples from an official source. I see that they merged the Banque de dépannage linguistique that we had used and a dictionnary, and the contents are accessible via a search bar only. Perhaps the best thing to do would be to **not replace** these links but to simply remove them and, where necessary, just propose the link towards the home page of the current website. If you open a draft pull request, I can contribute and ask Marie review and request changes, if she wants.   
How urgent is this issue @anisa-hawes ? Does it block the merges for other pull requests?

**Comment by anisa-hawes** [comment]
Ah, I see. Thank you, @spapastamkou. Yes, I noticed that the links appear to contain queries. 

I can remove the links, (perhaps excluding the first one?). I'll open a PR to do so. 

When you have a moment, I would be grateful for your time to check it through. Getting this perfect is not _urgent_ but removing the broken links will make it possible for us to move other PRs forwards. There is actually another broken link in our research pages which is blocking our way, and it was in the course of fixing that I found this problem 🙃
